pkgs/by-name/ma/mapserver/package.nix

```diff
@@ -81,6 +84,16 @@ stdenv.mkDerivation rec {
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
   ] ++ lib.optional withPython "-DWITH_PYTHON=ON";
 
+  passthru.updateScript = writeScript "update.sh" ''
+     #! ${stdenv.shell}
+     set -o errexit
+     set -x
+     # mapserver tags are: rel-MAJOR-MINOR-PATCH
+     # occasionnally, there are rel-MAJOR-MINOR-PATCH-betaN
+     version=$(${common-updater-scripts}/bin/list-git-tags --url=https://github.com/mapserver/mapserver | sed -E -n  "s/rel-([0-9])+-([0-9])+-([0-9])+$/\1.\2.\3/p" | sort --version-sort --reverse | head -n1)
```

Comment:
Please wrap this line to make it shorter.

Reply:
I get incorrect version value when testing update using

```
nix run github:Mic92/nix-update -- mapserver
```

dashes instead of dots.
```
git diff

@@ -33,7 +32,7 @@

 stdenv.mkDerivation rec {
   pname = "mapserver";
-  version = "8.2.2";
+  version = "8-2-2";

   src = fetchFromGitHub {
     owner = "MapServer";
```